specification/api-metrics-user.md

```diff
@@ -38,7 +36,7 @@ external systems.  Metric names conform to the following syntax:
 4. Subsequent characters must be belong to the alphanumeric characters, '_', '.', and '-'.
 
 Metric names belong to a namespace. The `component` label value of the associated `Meter` serves as its namespace,
-allowing the same metric name to be used in multiple libraries of code, unambiguously, within the same application.  
+allowing the same metric name to be used in multiple libraries of code, unambiguously, within the same application.
```

Comment:
two spaces at the end of a line create a line break without introducing a new paragraph and were added intentionally